File: Daily Challanges/2026-03-March/2026-03-01.py
def is_flat(arr):
    return all([type(item) != list for item in arr])

test1 = is_flat([1, 2, 3, 4])
test2 = is_flat([1, [2, 3], 4])
test3 = is_flat([1, 0, False, True, "a", "b"])
test4 = is_flat(["a", [0], "b", True])
test5 = is_flat([1, [2, [3, [4, [5]]]], 6])

check1 = True
check2 = False
check3 = True
check4 = False
check5 = False

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")

File: Daily Challanges/2026-03-March/2026-03-01.js
function isFlat(arr) {
  return arr.every((i) => !Array.isArray(i));
}

const test1 = isFlat([1, 2, 3, 4]);
const test2 = isFlat([1, [2, 3], 4]);
const test3 = isFlat([1, 0, false, true, "a", "b"]);
const test4 = isFlat(["a", [0], "b", true]);
const test5 = isFlat([1, [2, [3, [4, [5]]]], 6]);

const check1 = true;
const check2 = false;
const check3 = true;
const check4 = false;
const check5 = false;

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);
